evap/evaluation/tests/test_models_logging.py

```diff
@@ -147,3 +148,44 @@ def test_simultaneous_add_and_remove(self):
             self.evaluation.related_logentries().order_by("id").last().data,
             {"participants": {"add": [participant2.id], "remove": [participant1.id]}},
         )
+
+    def test_logging_m2m_reverse_changes(self):
+        participant = baker.make(UserProfile)
+
+        participant.evaluations_participating_in.add(self.evaluation)
+        self.assertEqual(
+            self.evaluation.related_logentries().order_by("id").last().data,
+            {"participants": {"add": [participant.id]}},
+        )
+
+        participant.evaluations_participating_in.remove(self.evaluation)
+        self.assertEqual(
+            self.evaluation.related_logentries().order_by("id").last().data,
+            {"participants": {"remove": [participant.id]}},
+        )
+
+        participant.evaluations_participating_in.add(self.evaluation)
+        participant.evaluations_participating_in.clear()
+        self.assertEqual(
+            self.evaluation.related_logentries().order_by("id").last().data,
+            {"participants": {"remove": [participant.id]}},
+        )
+
+        result_no_field_name = _m2m_changed(None, None, None, True, Evaluation, None)
+        self.assertEqual(result_no_field_name, None)
+
+    def test_logging_respects_unlogged_fields(self):
+        participant = baker.make(UserProfile)
+        original_unlogged_fields = self.evaluation.unlogged_fields
+
+        # Temporarily modify unlogged_fields to include 'participants'
+        with patch.object(
+            self.evaluation.__class__,
+            "unlogged_fields",
+            new=property(lambda self: original_unlogged_fields + ["participants"]),
+        ):
```

Comment:
Does this not work? If so, why?
```suggestion
        with patch.object(Evaluation, "unlogged_fields", new=["participants"]):
```